fsharp-backend/src/LibExecution/Shortcuts.fs

```diff
@@ -63,7 +68,7 @@ let rec toStringRepr (e : Expr) : string =
   | EList (_, exprs) ->
     let exprs = List.map r exprs |> String.concat "; "
     $"eList [{exprs}]"
-  | _ -> $"Bored now: {e}"
+  | _ -> $"Bored now: {e}" // should we finish this? TODO check usages
```

Comment:
No, this was purely for writing code from existing expressions, which I wanted to do for middleware. I think we'd use FSharpToExpr for this now. I haven't used this in forever, no objection to deleting it.